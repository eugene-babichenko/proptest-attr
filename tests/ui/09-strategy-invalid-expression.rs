use proptest::prelude::*;
use proptest_attr::proptest;

fn main() {
    basic_test();
}

#[proptest(strategy = "hello_world()")]
fn basic_test(_value: u8) -> Result<(), TestCaseError> {
    Ok(())
}
