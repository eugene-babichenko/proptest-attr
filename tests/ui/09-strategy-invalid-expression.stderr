error[E0425]: cannot find function `hello_world` in this scope
 --> $DIR/09-strategy-invalid-expression.rs:8:23
  |
8 | #[proptest(strategy = "hello_world()")]
  |                       ^^^^^^^^^^^^^^^ not found in this scope
